src/components/YourCompany/YourCompany.vue

```diff
@@ -5,32 +5,32 @@
       attach="#app"
     />
 
-     <div class="define-company-header">
-        <v-icon color="app-dk-blue">mdi-domain</v-icon>
-        <label class="define-company-title">Your Company</label>
+    <div class="define-company-header">
+      <v-icon color="app-dk-blue">mdi-domain</v-icon>
+      <label class="define-company-title">Your Company</label>
     </div>
 
-    <div class="section-container">
-      <v-layout row class="mx-0 mt-4">
-        <v-flex xs3>
-          <v-layout column>
-            <label><strong>Company Name</strong></label>
-            <v-flex md1 class="mt-1">
-              <v-chip v-if="companyNameChanges || (isAlterationView && hasBusinessNameChanged)"
-                      id="corrected-lbl"
-                      x-small label
-                      color="primary"
-                      text-color="white"
-              >
-                 {{editedLabel}}
-              </v-chip>
-            </v-flex>
-          </v-layout>
-        </v-flex>
+    <div class="section-container mt-4">
+      <!-- Company Name -->
+      <v-row no-gutters>
+        <v-col cols="3">
+          <label><strong>Company Name</strong></label>
+          <v-flex md1 class="mt-1">
+            <v-chip v-if="companyNameChanges || (isAlterationView && hasBusinessNameChanged)"
+                    id="corrected-lbl"
+                    x-small label
+                    color="primary"
+                    text-color="white"
+            >
+              {{editedLabel}}
+            </v-chip>
+          </v-flex>
+        </v-col>
 
+        <!-- Display Mode -->
         <template v-if="!isEditingNames">
-          <v-flex xs6 class="mt-n1">
-            <div class="company-name font-weight-bold">{{ companyName }}</div>
+          <v-col cols="7" class="mt-n1">
```

Comment:
Columns are:
3 for label
7 for content (not 6)
2 for buttons

Or:
3 for label
9 for edit block